plugins/jdg70/src/main/resources/server.xml

```diff
@@ -0,0 +1,399 @@
+<?xml version="1.0" ?>
+
+<server xmlns="urn:jboss:domain:4.0">
```

Comment:
I would also consider XPath modifications on the default config - storing only the list of modifications in RG sources, rather than the whole file.